### PR TITLE
Add latest frigate version to stats endpoints

### PR DESCRIPTION
**Pull request by NickM-27**
This is in order to support the efforts of https://github.com/blakeblackshear/frigate-hass-integration/issues/235, frigate will check github for the latest version tag once on startup so that services (and potentially frigate itself in the future) can be aware if there is an update. 

I found that the `stats_tracking` was the "cleanest" way I could think of to pull the value only once and have it be included in both http and mqtt calls. If there is a better solution happy to reorganize this 👍

### CODE REVIEW

**Comment by drinfernoo** [comment]
What if Frigate isn't restarted for an indeterminate amount of time? It will only check for updates whenever it is?

**Comment by NickM-27** [comment]
> What if Frigate isn't restarted for an indeterminate amount of time? It will only check for updates whenever it is?

I spent some time thinking about it, but I would rather not spend time / resources polling since this should only consider full releases and not betas which means there would be a lot of redundant checks relative to the number of checks that found a new version. And since it is just supporting a HASS entity and nothing too important. 

If this is deemed more important than I think it is, there's a few ways I can think to have it poll once a day.

File: frigate/stats.py [hunk 1]
```diff
@@ -13,11 +15,25 @@
 logger = logging.getLogger(__name__)
 
 
+def get_latest_version() -> str:
+    try:
+        raw = urlopen('https://api.github.com/repos/blakeblackshear/frigate/releases/latest')
```
Comment (blakeblackshear) [inline frigate/stats.py:20]: I have included the `requests` library as a part of integrating Frigate+. We should use that here too.

Comment (NickM-27) [reply frigate/stats.py:20]: Ah nice, that'll be cleaner

File: frigate/stats.py [hunk 2]
```diff
@@ -13,11 +15,24 @@
...
 
+def get_latest_version() -> str:
+    try:
+        response = requests.get('https://api.github.com/repos/blakeblackshear/frigate/releases/latest').json()
+
+        if response:
+            return response.get("tag_name", "unknown").replace("v", "")
+        else:
+            return "unknown"
+    except URLError:
```
Comment (blakeblackshear) [inline frigate/stats.py:26]: The requests library doesn't throw this exception to my knowledge. Non-200 responses don't throw exceptions. I think you want to use 
```python
if response.ok:
```

**Pull request merged**